src/main/java/com/mineinabyss/mobzy/listener/MobListener.kt

```diff
@@ -116,17 +108,19 @@ object MobListener : Listener {
 
         //create an item based on model ID in head slot if entity will be using itself for the model
         entity.with<Model> { model ->
+            //TODO any way to do this without NMS?
+            if (model.small) mob.toNMS().isBaby = true
```

Comment:
the subinterface `Ageable` has a `setBaby` method. Something like `(mob as? Ageable)?.setBaby()`